### PR TITLE
Reserve enum range starting at 0x96F0 for Arm

### PR DESCRIPTION
**Pull request by janharaldfredriksen-arm**

### CODE REVIEW

File: xml/gl.xml [hunk 1]
```diff
@@ -6959,7 +6963,7 @@ typedef unsigned int GLhandleARB;
      file) File requests in the Khronos Bugzilla, OpenGL project, Registry
      component. -->
 
-    <enums namespace="GL" start="0x96F0" end="99999" vendor="ARB" comment="RESERVED FOR FUTURE ALLOCATIONS BY KHRONOS">
+    <enums namespace="GL" start="0x9700" end="99999" vendor="ARB" comment="RESERVED FOR FUTURE ALLOCATIONS BY KHRONOS">
         <unused start="0x96F0" end="99999" comment="RESERVED"/>
```
Comment (NogginBops) [inline xml/gl.xml:6967]: This line should likely also be updated to `0x9700` instead of `0x96F0`.

Comment (janharaldfredriksen-arm) [reply xml/gl.xml:6967]: Indeed it should. Thanks :)

**Comment by pdaniell-nv** [comment]
@oddhack this is approved if you're happy with it.

**Comment by oddhack** [comment]
> @oddhack this is approved if you're happy with it.

For some reason my review comment wasn't cleared even though the PR was updated accordingly. Maybe the force push confused github? Anyway, LGTM now, merging.

**Comment by pdaniell-nv** [comment]
@oddhack was this merged? It appears to still be open so unsure if your last comment was accurate.

**Pull request merged**